Submitted By: BLFS Book <blfs-book@linuxfromscratch.org>
Date: 2003-10-02
Initial Package Version: 2.3.1-2
Origin: BLFS Book
Description: Change locations to match a standard LFS install.
--- install.cfg.orig	2003-04-03 11:33:47.000000000 -0600
+++ install.cfg	2003-04-03 11:36:26.000000000 -0600
@@ -30,7 +30,7 @@
 TWPOLICY="/etc/tripwire"
 
 # Tripwire manual pages are stored in TWMAN.
-TWMAN="/usr/man"
+TWMAN="/usr/share/man"
 
 # Tripwire database files are stored in TWDB.
 TWDB="/var/lib/tripwire"
@@ -48,7 +48,7 @@
 TWREPORT="${TWDB}/report"
 
 # This sets the default text editor for Tripwire.
-TWEDITOR="/bin/vi"
+TWEDITOR="/usr/bin/vi"
 
 # TWLATEPROMTING controls the point when tripwire asks for a password.
 TWLATEPROMPTING=false
@@ -85,7 +85,7 @@
 #####################################
 
 TWMAILMETHOD=SENDMAIL
-TWMAILPROGRAM="/usr/lib/sendmail -oi -t"
+TWMAILPROGRAM="/usr/sbin/sendmail -oi -t"
 
 #####################################
 # SMTP options
